addons/binding/org.openhab.binding.draytonwiser/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,429 @@
+    </thing-type>
+
+    <!-- Channel Types -->
+    <channel-type id="currentTemperature-channel">
+        <item-type>Number</item-type>
+        <label>Temperature</label>
+        <description>Current temperature in degrees celsius</description>
+        <category>Temperature</category>
+        <state readOnly="true" pattern="%.1f °C"></state>
+    </channel-type>
+    
+    <channel-type id="currentHumidity-channel">
+        <item-type>Number</item-type>
+        <label>Humidity</label>
+        <description>Current Humidity in %</description>
+        <state readOnly="true" pattern="%d %%">
+        </state>
+    </channel-type>
+    
+    <channel-type id="currentSetPoint-channel">
+        <item-type>Number</item-type>
+        <label>Set Point</label>
+        <description>Current set point in degrees celsius</description>
+        <category>Temperature</category>
+        <state readOnly="false" pattern="%.1f °C"></state>
+    </channel-type>
+    
+    <channel-type id="batteryVoltage-channel" advanced="true">
+        <item-type>Number</item-type>
+        <label>Battery Voltage</label>
+        <description>Current Battery Voltage for the device</description>
+        <state readOnly="true" pattern="%.1f V"/>
+    </channel-type>
+    
+    <channel-type id="demandPercent-channel">
+        <item-type>Number</item-type>
+        <label>Current Heat Demand</label>
+        <description>Current heat demand in %</description>
+        <state readOnly="true" pattern="%d %%"></state>
+    </channel-type>
+    
+    <channel-type id="heatRequest-channel">
+        <item-type>Switch</item-type>
+        <label>Currently Requesting Heat</label>
+        <description>Is this room requesting heat</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="heatingOverride-channel">
+        <item-type>Switch</item-type>
+        <label>Heating Override</label>
+        <description>The heating override button has been pressed</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="hotWaterOverride-channel">
+        <item-type>Switch</item-type>
+        <label>Hot Water Override</label>
+        <description>The hot water override button has been pressed</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="signalRSSI-channel" advanced="true">
+        <item-type>Number</item-type>
+        <label>Signal RSSI</label>
+        <description>The reported network signal RSSI</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="signalLQI-channel" advanced="true">
+        <item-type>Number</item-type>
+        <label>Signal LQI</label>
+        <description>The reported network signal LQI</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="signalStrength-channel" advanced="true">
```

Comment:
This is a perfect candidate to be replaced by a `system-channel`, see https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#channels

Please also have a look if you can map some of your other channels to system-channels and remove your definitions afterwards.